scripts/sync-from-prod.sh

```
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -euo pipefail
+cd "$(dirname "$0")/.."
+
+echo "==> Dumping production schema..."
+npx supabase db dump -f supabase/migrations/00000000000000_baseline.sql --linked
+
+if [[ "${1:-}" != "--schema-only" ]]; then
```

Shellcheck (0.11.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)